REQUIRE

```diff
@@ -1,8 +1,5 @@
 julia 0.5
+ColorTypes
 Images 0.5
-Colors 0.6
-ColorVectorSpace 0.1
+Distributions 0.12
 FixedPointNumbers 0.1
```

Comment:
We should also bump the FPN min version to 0.3; it now uses `Normed`, which is the new name in 0.3.